test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -1011,7 +1045,14 @@ NESTED_PROTO_TOSTRING(ExpressionAssignment, assignment, Field)
 NESTED_PROTO_TOSTRING(NameAssignment, assignment, Field)
 {
 	std::string assignment_str = NameToString(assignment.name());
-	assignment_str += " = " + ExpressionToString(assignment.value());
+	/*
+	 * There are cases when the same name is used for a value in
+	 * a table and then for a method: `wrap_val({a = {}}):a()`.
```

Comment:
Since `wrap_val` is gone, this comment should be updated.

Reply:
Removed.